telebot/modules/stickers.py

```diff
@@ -55,6 +56,26 @@ def get_sticker(update: Update, context: CallbackContext):
         update.effective_message.reply_text("Please reply to a sticker for me to upload its PNG.")
 
 
+def _get_pack_num_and_name(user: User, bot: Bot):
```

Comment:
Random nitpicks, return type ka bhi type hints

Reply:
I do intend to properly comment and add type hints everywhere once I get this to a point where people actually feel like neko chan is a better alternative to notamemebot

Reply:
Cool